### PR TITLE
Hide Miniview when the previewed window has focus.

### PR DESCRIPTION
**Pull request by dontfreakout**
This will hide Miniview when the previewed window has focus.

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -486,6 +488,17 @@ class Miniview {
         }
     }
 
+    _windowFocusMonitor(display) {
+        let activeWindow = display.get_focus_window();
+        //global.log(`miniview: _windowFocusMonitor   : display=${display} window=${activeWindow.get_title()}`);
+
+        if (this.lookupIndex(activeWindow) == this._winIdx) {
```
Comment (iamlemec) [inline extension.js:495]: I think you can get away with:
`if (activeWindow == this._metaWin) {`

**Comment by iamlemec** [comment]
Great idea, thanks! Just have that one tweak, and ready to merge after that.

File: extension.js (hunk 1 above)
Comment (dontfreakout) [reply extension.js]: You are right, didn't notice the **_metaWin** property :-)

**Comment by dontfreakout** [comment]
Besides **this._metaWin** tweak, I added the possibility to disable this feature from settings. 
_prefs.js part is not in this commit (still WIP)_

**Comment by iamlemec** [comment]
Excellent. I'll merge now and add in a checkbox on prefs.

Also, trying to decide if this should be on by default. Just a bit worried about a first time use getting confused.

**Comment by iamlemec** [comment]
Just pushed a slight refactor. Now the display logic all runs through `realizeMiniview` and we just call that when things change. Also added in a toggle to the prefs window. Would be great if you could test. Thanks!

**Pull request merged**